### DISCUSSION TITLE
Is it safe to disable System Navigation?

### DISCUSSION THREAD

**Discussion by ImLimitedEd** [celzero/rethink-app#1116] [Q&A]
I couldn't find the answer on the internet. Is it safe to disable it? or it'll cause a problem?
Here's the screenshot. https://ibb.co/17GT93z
https://ibb.co/QNXGYm4
(I don't know why but GitHub doesn't allow me to upload image).

**Comment by ignoramous** [comment]
Sorry, what's "System Navigation"?
